specification.md

```diff
@@ -2446,7 +2608,7 @@ it with its `object` type which has the following properties:
 | Parameter | Description | Type | Required |
 | --- | --- | --- | --- |
 | refName | Name of the referenced [function](#Function-Definition) | string | yes |
-| arguments | Arguments to be passed to the referenced function | object | no |
+| arguments | Arguments to be passed to the referenced function | object | only for GraphQL functions |
```

Comment:
could you please change
"only for GraphQL functions"
to 
"yes if function type is graphql, otherwise no"